docs/reference/ilm/ilm-tutorial.asciidoc

```diff
@@ -20,27 +20,42 @@ and still get the most out of your data.
 
 To automate rollover and management of time-series indices with {ilm-init}, you:
 
-. <<ilm-gs-create-policy, Create a lifecycle policy>> with the {ilm-init} put policy API.
+. <<ilm-gs-create-policy, Create a lifecycle policy>> that defines the appropriate
+phases and actions.  
 . <<ilm-gs-apply-policy, Create an index template>> to apply the policy to each new index.
 . <<ilm-gs-bootstrap, Bootstrap an index>> as the initial write index.
-. <<ilm-gs-check-progress, Verify indices are moving through the lifecycle phases>>
-as expected with the {ilm-init} explain API.
+. <<ilm-gs-check-progress, Verify indices are moving through the lifecycle phases>> 
+as expected.
+
+For an introduction to rolling indices, see <<index-rollover>>. 
+
+NOTE: {fb} includes a default ILM policy that initiates the rollover action when
```

Comment:
`{fb}` is rendering literally here. Not sure what it's supposed to refer to.

Reply:
Supposed to be Filebeat. :-)